src/include/sof/common.h

```diff
@@ -113,6 +113,9 @@
 #define container_of(ptr, type, member) \
 	({const typeof(((type *)0)->member)*__memberptr = (ptr); \
 	(type *)((char *)__memberptr - offsetof(type, member)); })
+#define attr_container_of(ptr, type, member, attr) \
```

Comment:
I assume this aligns with Zephyr ?

Reply:
ok, we will need to align these at some point, but Zephyr would need to have the same helpers first. @andyross fyi.